quickstarts/notification-channels/email-notifications/config.yml

```diff
@@ -0,0 +1,30 @@
+name: email-notifications
+
+title: Email Notifications
+
+description:  |
+  Send your New Relic alert notifications via email.
+  Check out our documentation to set up an email notification channel and provide fast and consistent ways for the right personnel to be notified about incidents
+
+summary: |
+  Send your New Relic alerts via email
+logo: logo.png
+level: New Relic
+
+authors:
+  - Matan Moser (New Relic)
+  - Ismail Azam (New Relic)
+
+keywords:
+  - email
+  - notifications
+  - AIOps
+  - Incident Intelligence
+  - Destinations
+
+documentation:
```

Comment:
Won't this add a tile to the data source tab? Which I thought we agreed would confuse the user and the idea was to just link to these doc URLS from within the markdown description.